frontend/src/app/services/covid.service.ts

```diff
@@ -38,4 +38,8 @@ export class CovidService {
       });
   }
 
+  public getProvinceSituationPerDate(): Promise<GeneralSituationDaily[]> {
```

Comment:
The response type is not GeneralSituationDaily[] anymore. You can merge the master in your branch and use the type AreaResponse.